addons/stock/static/src/stock_forecasted/stock_forecasted.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates id="template" xml:space="preserve">
+    <div t-name="stock.Forecasted" class="o_action o_stock_forecasted_page" owl="1">
```

Comment:
```suggestion
    <div t-name="stock.Forecasted" class="o_action" owl="1">
```
`o_stock_forecasted_page` is set too high on the DOM, it changes even the breadcrumbs links. Need to set it only on the content of the report, not the Control Panel